container/docker/handler.go

```diff
@@ -294,32 +294,8 @@ func (self *dockerContainerHandler) GetStats() (*info.ContainerStats, error) {
 }
 
 func (self *dockerContainerHandler) ListContainers(listType container.ListType) ([]info.ContainerReference, error) {
-	if self.name != "/docker" {
```

Comment:
Coincentally I've just run cadvisor on Ubnutu 14.04 & it does actually use `/docker`...

```
condescending_bhabha (/docker/f5a39e9c98788da89c050ca45d266ce0ddc31bd0a7e3ecbc618e166c17ade17d)
```